diff --git a/node_modules/cordova-plugin-firebasex/www/firebase.js b/node_modules/cordova-plugin-firebasex/www/firebase.js
index 1f05423..b950d98 100644
--- a/node_modules/cordova-plugin-firebasex/www/firebase.js
+++ b/node_modules/cordova-plugin-firebasex/www/firebase.js
@@ -343,7 +343,7 @@ exports.authenticateUserWithMicrosoft = function (success, error, locale) {
   exec(success, error, "FirebasePlugin", "authenticateUserWithMicrosoft", [locale]);
 };
 
-exports.authenticateUserWithFacebook = function (accessToken, success, error,) {
+exports.authenticateUserWithFacebook = function (accessToken, success, error) {
     exec(success, error, "FirebasePlugin", "authenticateUserWithFacebook", [accessToken]);
 };
 
